### ISSUE TITLE
BlazorWasm build benchmark broken

### ISSUE THREAD

**Issue by sebastienros** [dotnet/aspnetcore#50821]
https://dev.azure.com/dnceng/internal/_build/results?buildId=2270625&view=logs&j=aea04f6b-9203-5996-a2c9-49c606d4b706&t=ee2bb378-1359-50a5-a5e1-3ebfb43fc36e

Probably a template has changed and this benchmark can't find one of the files it was trying to update to benchmark how it impact the build perf.

**Comment by sebastienros** [comment]
Closing as duplicate of https://github.com/dotnet/aspnetcore/issues/49760

You may want to reassign the work on the other issue

**Comment by sebastienros** [comment]
Not a duplicate actually

**Comment by lewing** [comment]
Looks like the template moved the `Pages` directory to `Components/Pages`

**Comment by MackinnonBuck** [comment]
It looks like @lewing fixed this in https://github.com/aspnet/Benchmarks/pull/1916 (thanks!). Closing this out as completed.

**Comment by lewing** [comment]
It will move back again once flow in 9.0 is unblocked
